test_xdb/models/under_test/regexp_count_test.sql

```diff
@@ -0,0 +1,10 @@
+WITH
+source_data AS (
+    SELECT
+        'abba is a great band' AS false_statement
```

Comment:
would like to see a few more regex cases in here that account for `/` in snowflake.

Reply:
added 2 good ones